### PR TITLE
[Doris On ES]Add documentation for doris on es

### PR DESCRIPTION
**Pull request by wuyunfeng**

### CODE REVIEW

File: docs/documentation/cn/administrator-guide/doris-on-es/doris-on-es.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# Doris-On-ES使用手册
```
Comment (imay) [inline docs/documentation/cn/administrator-guide/doris-on-es/doris-on-es.md]: put it to  docs/documentation/cn/extending-doris

**Review by imay — CHANGES_REQUESTED** [review]

File: docs/documentation/cn/administrator-guide/extending-doris/doris-on-es.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+"type" = "doc"
+);
+```
+
+参数说明：
+
+参数 | 说明
+---|---
+host | ES集群连接地址，可指定一个或多个，Doris通过这个地址获取到ES版本号、index的shard分布信息
+user | 开启basic认证的ES集群的用户名，需要确保该用户有访问: /、/_cluster/state、_nodes/http等路径权限
```
Comment (morningman) [inline docs/documentation/cn/administrator-guide/extending-doris/doris-on-es.md]: 这里 下滑线 _ 需要加 反斜杠 \ ，否则会被认为是斜体

Comment (morningman) [reply docs/documentation/cn/administrator-guide/extending-doris/doris-on-es.md]: 下同

**Review by morningman — CHANGES_REQUESTED** [review]

File: docs/documentation/cn/administrator-guide/extending-doris/doris-on-es.md (hunk 2 above)
Comment (wuyunfeng) [reply docs/documentation/cn/administrator-guide/extending-doris/doris-on-es.md]: hi，morningman. please review this change

File: docs/documentation/cn/administrator-guide/doris-on-es/doris-on-es.md (hunk 1 above)
Comment (wuyunfeng) [reply docs/documentation/cn/administrator-guide/doris-on-es/doris-on-es.md]: OK, Please review again.

**Review by morningman — DISMISSED** [review]

File: docs/documentation/cn/extending-doris/doris-on-es/doris-on-es.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+select * from es_table where k1 > 1000 and k3 ='term' or k4 like 'fu*z_'
+```
+
+#### 扩展的esquery sql语法
+通过`esquery`函数将一些无法用sql表述的ES query如match、geoshape等下推给ES进行过滤处理，`esquery`的第一个列名参数用于关联`index`，第二个参数是ES的基本`Query DSL`的json表述，使用花括号`{}`包含，json的`root key`有且只能有一个，如match、geo_shape、bool等
+
+match查询：
+
+```
+select * from es_table where esquery('k4', '{
```
Comment (imay) [inline docs/documentation/cn/extending-doris/doris-on-es/doris-on-es.md]:
```suggestion
select * from es_table where esquery(k4, '{
```

File: docs/documentation/cn/extending-doris/doris-on-es/doris-on-es.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+  `k1` bigint(20) COMMENT "",
+  `k2` datetime COMMENT "",
+  `k3` varchar(20) COMMENT "",
+  `k4` varchar(100) COMMENT "",
+  `k5` float COMMENT ""
+) ENGINE=ELASTICSEARCH
+PARTITION BY RANGE(`id`)
+()
+PROPERTIES (
+"host" = "host",
```
Comment (imay) [inline docs/documentation/cn/extending-doris/doris-on-es/doris-on-es.md]: host:port

File: docs/documentation/cn/extending-doris/doris-on-es/doris-on-es.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+
+## 其他说明
+
+1. ES的版本要求
+
+    ES主版本大于5，ES在2.x之前和5.x之后数据的扫描方式不同，目前支持5.x之后的
+2. 是否支持X-Pack认证的ES集群
+
+    支持所有使用http basic认证方式的ES集群
```
Comment (imay) [inline docs/documentation/cn/extending-doris/doris-on-es/doris-on-es.md]:
```suggestion
    支持所有使用HTTP basic认证方式的ES集群
```

**Review by imay — APPROVED** [review]

**Pull request merged**